p2p/peer_tracker.go

```diff
@@ -24,7 +26,7 @@ var (
 	// otherwise it will be removed on the next GC cycle.
 	maxAwaitingTime = time.Hour
 	// gcCycle defines the duration after which the peerTracker starts removing peers.
-	gcCycle = time.Minute * 30
+	gcCycleDefault = time.Minute * 1
```

Comment:
This could have some interesting implications so we should definitely test this out on Robusta before releasing.